web-api/terraform/template/elasticsearch.tf

```diff
@@ -49,3 +49,27 @@ resource "aws_elasticsearch_domain" "efcms-search" {
     log_type                 = "ES_APPLICATION_LOGS"
   }
 }
+
+resource "aws_elasticsearch_domain" "efcms-app-logs" {
+  domain_name           = "efcms-app-logs-${var.environment}"
```

Comment:
Maybe `efcms-logs-${var.environment}` instead? This name seems to indicate that the index only contains logs from `app.${var.dns_domain}` — but I think we’re planning on dropping all the logs in here (correct me if I’m wrong!).

Reply:
good point, I agree that the use of "app" sounds restrictive. Will change!

Reply:
what about `logs.${var.zone_name}`?